core/src/test/java/forklift/consumer/injection/ConsumerTest.java

```diff
@@ -224,6 +229,49 @@ public void testHeadersAndProperties() {
         assertEquals(ec.producer, "testing");
     }
 
+    // For the next two tests, we are testing named dependency injection.
```

Comment:
Minor Opinion:  This comment seems a little brittle in that code methods could move